.github/workflows/address-sanitizer.yml

```diff
@@ -27,17 +27,17 @@ jobs:
       - name: Install Dependencies
         run: |
           if [ -f /etc/fedora-release ]; then
-            dnf -y install git clang gcc automake libtool pkgconf-pkg-config \
-              autoconf-archive openssl-devel openssl diffutils expect \
+            dnf -y install git clang gcc pkgconf-pkg-config meson \
+              openssl-devel openssl diffutils expect \
```

Comment:
This is not a note for the meson change, but generic note to the way how the github actions are used here now.

When we were moving the CI of OpenSC from something previous to GH actions, I wanted to keep the bare minimum in the GH yml files and keep most of the logic in separate shell scripts, such as:

https://github.com/OpenSC/OpenSC/blob/master/.github/setup-linux.sh

This avoids when we change some dependencies to do it in half a dozen of yaml files.

This does not have to be changed as this PR, but I wanted to mention it here if you would find it useful, we can do it later.